lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -4562,6 +4591,19 @@ TypeSystemClang::AddConstModifier(lldb::opaque_compiler_type_t type) {
   return CompilerType();
 }
 
+CompilerType
+TypeSystemClang::AddPtrAuthModifier(lldb::opaque_compiler_type_t type,
+                                    uint32_t payload) {
+  if (type) {
```

Comment:
Could change these to early-return (which [LLVM style guide recommends](https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code)). I.e.:
```
if (!type)
  return {};

...
```

This applies to the other functions above as well

Reply:
Yes, this would be much nicer, thanks for suggestion! Since almost all the functions here use this "wrong" pattern (and also some others like else after return), I suggest to leave things here "as is" and open a subsequent PR changing style in all these functions after the current PR is merged. Please let me know your thoughts on this.

Reply:
Yup sounds good. I don't mind sticking to the style of the file for now